### PR TITLE
docs: add `CDNJS` example

### PR DESCRIPTION
**Pull request by Torres-ssf**
- Closes: https://github.com/FuelLabs/fuels-ts/issues/1397

### CODE REVIEW

File: apps/docs/src/getting-started.md [hunk 1]
```diff
@@ -62,6 +67,28 @@ function App() {
 export default App;
 ```
 
+## Browser
+
+For a quick test or just playing around, you can load it in your Web Apps straight from our CDN.
```
Comment (danielbate) [inline apps/docs/src/getting-started.md:72]: Is it worth noting here that this is only supported from `0.73.0`?

**Review by danielbate — APPROVED** [review]

File: apps/docs/src/getting-started.md (hunk 1 above)
Comment (arboleya) [reply apps/docs/src/getting-started.md:72]: I thought the same, but the version is self-contained in the snippet URL.


version

File: apps/docs/src/getting-started.md [hunk 2]
```diff
@@ -62,6 +67,28 @@ function App() {
 export default App;
 ```
 
+## Browser
```
Comment (arboleya) [inline apps/docs/src/getting-started.md:70]:
```suggestion
## CDN Usage (browser only)
```

**Review by arboleya — COMMENTED** [review]
We should have provided a minified version, but that can be done in another PR.
 - https://github.com/FuelLabs/fuels-ts/issues/1728

**Review by danielbate — APPROVED** [review]

**Pull request merged**